streams/ru.m3u

```diff
@@ -248,3 +248,77 @@ http://stream0.tv41.ru/live.m3u8
 https://live-echotv.cdnvideo.ru/echotv/echotv.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="Yuvelirochka.ru",Ювелирочка ТВ (576p)
 https://live-uvelirochka.cdnvideo.ru/uvelirochka/uvelirochka_720p3/playlist.m3u8
+#EXTINF:-1 tvg-id="",TVOE TV HD
+http://tvoetv.space:8080/tvoetv/index.m3u8
+#EXTINF:-1 tvg-id="",Кино 1 International
+http://kino-1.catcast.tv/content/38617/index.m3u8
+#EXTINF:-1 tvg-id="",КИНО-1 INTERNATIONAL
+https://autopilot.catcast.tv/content/38617/index.m3u8
+#EXTINF:-1 tvg-id="",Кино 1 International HD
+http://213.172.73.132:8081/nimble_live/kino_1_int/playlist.m3u8
+#EXTINF:-1 tvg-id="",ViP COMEDY HD
+http://dmi3y-tv.ru/hls/okko/CH_ViPCOMEDYHD.m3u8
```

Comment:
We have that, it's nothing more than a mirror
https://github.com/iptv-org/iptv/blob/f37fffc95ddad37a0c4d7e6e73fab3b3d6b6ccc7/streams/ru_okkotv.m3u#L63C1-L63C59